protocol/proofs/pdg-no-proposer-play.md

```diff
@@ -0,0 +1,51 @@
+# Limit PermissionedDisputeGame Play to Challenger
+
+## Context
+
+The `PermissionedDisputeGame` is a version of the `FaultDisputeGame` that limits the parties that
+are allowed to execute moves within the game. The `PermissionedDisputeGame` is designed to act as
+either a fallback for the `FaultDisputeGame` or an independent game type if a chain is not using
+the permissionless `FaultDisputeGame` model. The `PermissionedDisputeGame` is intended to mirror
+the behavior of the legacy `L2OutputOracle` whereby an authorized `Proposer` can create proposals
+and an authorized `Challenger` can remove those proposals.
+
+## Problem Statement
+
+The `PermissionedDisputeGame` currently allows *both* the `Proposer` and the `Challenger` to
+participate in the game and create moves. The original intent of this behavior was to have the
+`PermissionedDisputeGame` act as a fully-fledged allowlisted dispute game. However, a number of
+changes to product needs and security modeling have popped up.
```

Comment:
By allowlisted, I assume you mean that the set of allowed players can be arbitrarily expanded, rather than fixed at 2.

Reply:
Correct